README.md

```diff
@@ -150,7 +150,13 @@ $ ./start.sh
 
 ##### Python
 
-**:warning:  As `python2` has reached its end of life, only `python3` is supported.**
+**:warning: As `python2` has reached its end of life, only `python3` is supported.**
+
+If you are running python 3.9, you may need to run the following command in your terminal in order to install SSL certificates:
+
+```
+open /Applications/Python\ 3.9/Install\ Certificates.command
```

Comment:
its pretty rare in the industry that people use the system version of python (the one installed in `/Applications...`). Most people use `pyenv` or something like that. How do you do this with `pyenv`?